### PR TITLE
Fix Log4j vulnerability

### PR DESCRIPTION
**Pull request by emilva**
There is a critical vulnerabiltiy in Log4j, more information here: https://www.lunasec.io/docs/blog/log4j-zero-day/

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
     <properties>
         <version.slf4j>1.7.32</version.slf4j>
-        <version.log4j2>2.14.1</version.log4j2>
+        <version.log4j2>2.15.1</version.log4j2>
```
Comment (chriswk) [inline pom.xml:10]: There's no 2.15.1 yet, I assume you meant 2.15.0 :)
```suggestion
        <version.log4j2>2.15.0</version.log4j2>
```

**Review by chriswk — CHANGES_REQUESTED** [review]

**Comment by emilva** [comment]
Yes, you are correct! :)

**Review by chriswk — APPROVED** [review]
Thank you!
:+1:

**Comment by barkanido** [comment]
Can this be also backported?

**Pull request merged**